docs/versioning/versioning-faq.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+nav_order: 3
+title: Versioning FAQ
+has_children: false
+parent: Versioning
+---
+
+# Versioning Frequently-asked-questions
```

Comment:
May need a "why would I need a new version?" so a new user can understand

Reply:
added in central faq